tasks/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -165,22 +165,24 @@ spec:
         echo -n "${mustPublishIndexImage}" | tee "$(results.mustPublishIndexImage.path)"
         echo -n "${fbc_opt_in}" | tee "$(results.isFbcOptIn.path)"
 
-        # Whether the index image will be published
+        jq '.reason // "Unset"'  <<< "${conditions}" | tee "$(results.requestReason.path)"
+        jq '.message // "Unset"' <<< "${conditions}" | tee "$(results.requestMessage.path)"
+
+        jq -r '.indexImageDigests' <<< "${results}" |  tee "$(results.indexImageDigests.path)"
+
+        jq -r '.iibLog' <<< "${results}"
+        RC="$(jq -r '.exitCode' <<< "${results}")"
+
+        # Summarize what happened for the human user.
         if [ "$mustPublishIndexImage" = "true" ]; then
           echo "Index image will be published."
         elif [ "$fbc_opt_in" = "false" ]; then
-          echo "Index image will not be published because FBC opt-in is set to false in Pyxis."
+          echo "Index image will not be published because fbc_opt_in is set to false in Pyxis."
+          echo "If this is the first time you are releasing, make sure you request fbc_opt_in in Pyxis."
```

Comment:
could there be a link to WHERE pyxis is, or WHERE to  apply this configuration? You seem to be assuming news knows what to do and where to do it :(

Reply:
It's covered here in the downstream docs: https://konflux.pages.redhat.com/docs/users/releasing/preparing-for-release.html#publishing-a-fbc-graph

> Before you release your FBC component to production index for the first time
[..]
> - Identify all unique bundle image repositories referenced in the FBC fragment. This can be achieved with a command like opm render $(params.fbcFragment) | jq -r 'select(.schema == "olm.bundle") | "\(.image)" | split("@")[0]' |uniq
> - Post a message in [#rel-eng](https://redhat.enterprise.slack.com/archives/C03CFJBGRTK) tagging @konflux-releng requesting these bundle(s) to be opted into FBC.

But not sure if we want to link the doc directly.

@nickboldt any suggestions?

Reply:
why not? if this is RH internal process, why not make it easier for RH internal people to find and follow it? A vague call to action is like tagging your name on a wall in a dark alley. Sure, some people might understand who did it and why, but everyone else will just think, "what does this mean?"

Reply:
also what should be put in `params.fbcFragment` ? That looks like a snipper of tekton code, not something that I, a newbie user of Konflux onboarding my FBC to release with an RPA the first time, can run locally to get the list to share with rel eng folks.

Maybe the process could remove the human element, and there could be a PR or MR generated with the information required, in a repo that would automatically hit the eyes of the konflux-releng people? I'm thinking in https://gitlab.cee.redhat.com/releng/konflux-release-data/-/merge_requests/ or something

Reply:
Do we really want links to internal systems in public tasks?

Reply:
>Sure, some people might understand who did it and why, but everyone else will just think, "what does this mean?
Isn't that the point of documentation? More than tekton task error messages?

Reply:
> also what should be put in params.fbcFragment

I don't see `params.fbcFragment` anywhere in the diff. How is it relevant here? It certainly looks like the syntax for a tekton parameter. If a newbie user of Konflux wants to go look at tekton task definitions, why would they not go find tekton parameters and instead find human readable stuff to run locally? This is a tekton task, not the docs repo

Reply:
> This can be achieved with a command like opm render $(params.fbcFragment) ...

it was in this [comment](https://github.com/konflux-ci/release-service-catalog/pull/621#discussion_r1804755309).

Reply:
you have a valid point about not linking to internal docs from public tasks. My point is more about UX -- make things simpler to debug and your users will be happier :D

Reply:
Yeah, agreed for sure. I just am not sure of what the line is between duplicating user documentation in tekton task messages. We definitely want some, quite sure we don't want all of it, not sure exactly how much

Reply:
My opinion - this PR is right on that line. If people see "make sure you request fbc_opt_in in Pyxis." and they didn't already see it in the docs, they'll find their way next to #konflux-users and the support group who will sort it out more quickly.

Reply:
It looks good to me. @nickboldt ?

Reply:
well, the call to action "make sure you request fbc_opt_in in Pyxis" is still a :confused-doggo: thing until you go find people in the internal slack channels but I guess it's better than before since you can at least ask an informed question like "hey konflux experts, how can I request fbc_opt_in in Pyxis for my upcoming initial GA?" 

so +1.